app/controllers/sword_controller.rb

```diff
@@ -16,44 +16,34 @@ def deposit
     # use @collection.parser, which will need to be changed to
     # @collection.endpoint, i.e. will need to change attribute in
     # collection model from parser to endpoint
-    @endpoint = get_endpoint(@collection_slug,
-                             @depositor_user_id)
-    @path_to_deposit_contents = Sword::Util::unzip_deposit_file request
+    @endpoint = Sword::Endpoints::Endpoint.get_endpoint(@collection_slug,
```

Comment:
since the endpoints module determines the type of endpoint here, do you need to import the specific endpoints in lines 2-4 of this file?